.github/workflows/helm-test.yaml

```
@@ -0,0 +1,31 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Run chart-testing (lint)
+        id: lint
+        uses: helm/chart-testing-action@v1.0.0-rc.1
+        with:
+          command: lint
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0-rc.1
+        # Only build a kind cluster if there are chart changes to test.
+        if: steps.lint.outputs.changed == 'true'
+        with:
+          version: v0.8.0
+
+      - name: Run chart-testing (install)
+        uses: helm/chart-testing-action@v1.0.0-rc.1
+        with:
+          command: install
```

I'm not sure the correct way to have GitHub add the missing new-line at EOF.
```suggestion
          command: install

```
Assuming you are editing your PR locally and pushing (force is ok), a normal text editor should automatically add the missing character if you open, edit, and save the file.

The GitHub editor will also let you fix this.

---

Thanks! See b9208cfc219b779f5bdaaac372fe5fa6a5ee1f1f (for both fixes — clicking "commit suggestion" in the GitHub web UI on the first also automatically fixed the second issue with the missing new-line).